api/manifest/manifest.go

```diff
@@ -153,6 +154,14 @@ func buildSync(source, tag string, cache bool) error {
 		args = append(args, "--no-cache")
 	}
 
+	if dockerfile != "" {
+		args = append(args, "-f")
```

Comment:
was going for speed with Buffer.WriteString instead, but I guess that's an overkill during the build step.  I'll change it